### ISSUE TITLE
Mention hacktoberfest as a good period to invite your user to contribute

### ISSUE THREAD

**Issue by Okazari** [zenika-open-source/open-source-promotion-cheat-sheet#47]
**I would like to ...**

- [X] to add content

**What is the section concerned ?**
3. Keep your users

**What is the point concerned ?**
Invite users to contribute

**What would you like to add or modify ?**
I think we could add a mention to the Hacktoberfest as a good time of the year to encourage contribution on the project :)

**Comment by kefranabg** [comment]
👍

**Comment by Seniru** [comment]
Agree, also adding the `hacktoberfest` label would help you to attract more crowd. I think this idea fits to the **Keep you users** section

**Comment by jeanphi-baconnais** [comment]
Hello, I just added this label on the project.

I think we can close this issue.
